### ISSUE TITLE
Bug: Sound browser filter category keyboard navigation

### ISSUE THREAD

**Issue by manodrum** [earsketch/earsketch-webclient#896]
Bug in sound browser filter navigation: If I enter to get into artists and then escape to get to the top level of filter categories, tab to the next category, and hit enter, my keyboard focus does not go to the options within the new category dropdown and I cannot tab through them. I have to escape again and enter again to get focus. 

There is a race condition because of activeButtonRef shared by all filter options
